src/bereikbaarheid/models.py

```diff
@@ -63,7 +63,7 @@ class Meta:
     link_nr = models.IntegerField(help_text="linknr")
     name = models.CharField(max_length=255, blank=True, null=True)
     e_type = models.CharField(max_length=255, blank=True, null=True)
-    verkeersbord = models.CharField(max_length=255, blank=True, null=True)
+    verkeersbord = models.IntegerField()
```

Comment:
In de migratie geef je `default=0` mee als kwarg. Heeft het zin om dat hier ook toe te voegen, of is dat impliciet?

Reply:
alleen toegevoegd omdat de db al data bevat, slechts eenmalig vanwege db consistentie.